src/SimpleExcelReader.php

```diff
@@ -233,7 +233,7 @@ public function headerRowFormatter(callable $callback)
 
     protected function trim(string $header): string
     {
-        return call_user_func_array('trim', array_filter([$header, $this->trimHeaderCharacters]));
+        return call_user_func_array('trim', array_merge([$header], array_filter([$this->trimHeaderCharacters])));
```

Comment:
I tried to follow the current "`call_use_func_array`" pattern but I think we can simplify it with the following:
```php
return trim($header, $this->trimHeaderCharacters);
```